docs/content/preview/troubleshoot/nodes/check-stats.md

```diff
@@ -36,23 +38,23 @@ Generally, common tools like `top` or `iostat` may be useful.
 
 ### Auditd
 
-If `top` reports high CPU usage for the `auditd` process, it may have some rules auditing some system calls frequently used YugabyteDB which can significantly affect performance. You can try temporarily disabling `audit` by running (on each YugabyteDB node).
+If `top` reports high CPU usage for the `auditd` process, it may have rules auditing some system calls frequently used YugabyteDB which can significantly affect performance. You can try temporarily disabling `audit` by running the following command on each YugabyteDB node:
 
 ```sh
-$ auditctl -e 0
+auditctl -e 0
 ```
 
-and check if this improves performance.
+Then you would check if this improves performance.
 
-To re-enable it afterwards, run:
+To re-enable `audit` afterwards, run the following command:
 
 ```sh
-$ auditctl -e 1
+auditctl -e 1
 ```
 
 ## YugabyteDB processes state
 
-YugabyteDB provides web endpoints where the current state of each process is aggregated. This includes logs, flags as well as memory, disk, and network usage metrics. Additionally, it provides dedicated metrics endpoints for YCQL and, respectively, Redis requests.
+YugabyteDB provides the following web endpoints where the current state of each process is aggregated. This includes logs, flags, as well as memory, disk, and network usage metrics. Additionally, it provides dedicated metrics endpoints for YCQL and, respectively, Redis requests:
```

Comment:
Not YSQL?

Reply:
Yes, I dont' know why there's nothing about YSQL.
It's out of scope for this ticket, so I'll create a new ticket to look into this stuff.